README.md

```diff
@@ -104,7 +73,8 @@ Give a :star: if this project helped you!
 ## :page_facing_up: License
 
 Copyright © 2019 [Monogramm](https://github.com/Monogramm).<br />
-This project is [AGPL v3](uri_license) licensed.
+This project is [MIT](uri_license) licensed.
```

Comment:
No, the license should be AGPL v3 by default, not MIT.

Also, why was the `LICENSE` file removed ??